### PR TITLE
Support Loadable wallet

### PR DESCRIPTION
**Pull request by scottphc**
- Support Loadable wallet
- Add a new optional property `propertyName` for multi-chain wallet which may have different brand name with window property name, such as `window.bloctoAptos`
- Implement multi signature verification

### CODE REVIEW

File: .changeset/forty-toys-dress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@aptos-labs/wallet-adapter-core": patch
+---
+
+Support Lodable wallet
```
Comment (0xmaayan) [inline .changeset/forty-toys-dress.md:5]: could you also add the implementation for signAndVerify change?

File: packages/wallet-adapter-core/src/WalletCore.ts [hunk 2]
```diff
@@ -329,7 +332,40 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
       let verified = false;
       if (Array.isArray(response.signature)) {
         // multi sig wallets
-        // TODO - implement multi sig wallets
+        const { fullMessage, signature, bitmap } = response
```
Comment (0xmaayan) [inline packages/wallet-adapter-core/src/WalletCore.ts:335]: thank you!

**Review by 0xmaayan — COMMENTED** [review]
Thank you for your contribution! 

Regarding the multi-chain wallet support, there is another suggestion on how we can go with it here https://github.com/aptos-labs/aptos-wallet-adapter/issues/57 , curios what are your thoughts

File: .changeset/forty-toys-dress.md (hunk 1 above)
Comment (scottphc) [reply .changeset/forty-toys-dress.md:5]: updated

**Review by 0xmaayan — APPROVED** [review]

**Pull request merged**